app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js

```diff
@@ -174,6 +187,36 @@ define([
                 block: 'center',
                 inline: 'nearest'
             });
-        }
+        },
+
+        /**
+         * Set image preview keyboard navigation listener
+         */
+        setNavigationListener: function () {
+            var imageIndex, endIndex, key,
+                startIndex = 0,
+                imageColumnSelector = '.masonry-image-column',
+                adobeModalSelector = '.adobe-stock-modal',
+                imageGridSelector = '.masonry-image-grid';
+
+            $(document).on('keydown', function(e) {
+                key = keyCodes[e.keyCode];
+                endIndex = $(imageGridSelector)[0].children.length - 1;
+
+                if($(this.previewImageSelector).length > 0) {
+                    imageIndex = $(this.previewImageSelector)
+                        .parents(imageColumnSelector)
+                        .data('repeatIndex');
+                }
+
+                if($(adobeModalSelector).hasClass('_show')) {
+                    if(key === 'pageLeftKey' && imageIndex !== startIndex) {
+                        $(this.previewImageSelector + ' .action-previous').click();
+                    } else if (key === 'pageRightKey' && imageIndex !== endIndex) {
+                        $(this.previewImageSelector + ' .action-next').click();
```

Comment:
Shouldn't next/prev methods of this component be called directly instead of triggering a click?

Reply:
Didn't realize `displayedRecord` property can be used for that purpose. Fixed it now, thank you!